docs/user-guide/aws-examples/eks.rst

```diff
@@ -0,0 +1,215 @@
+       - ReadWriteMany
+     storageClassName: efs-sc
+     resources:
+       requests:
+         storage: 150Gi
+   EOF
+
+This PVC requests 150GiB of storage with ReadWriteMany access mode from the efs-sc StorageClass.
+
+Dask cluster creation:
+----------------------
+
+Please refer to kubernetescurator.rst for instructions on creating a Docker secret to utilize the NeMo image and upload data to the PVC created in the previous step.
+
+Python environment can be setup by executing the following commands:
+
+.. code-block:: bash
+
+    python3 -m venv venv
+    source venv/bin/activate
+
+    pip install dask_kubernetes
+    pip install kubernetes
+
+The environment to run the provided scripts needs only dask-kubernetes and kubernetes packages.
+
+.. code-block:: bash
+
+  python3 examples/k8s/create_dask_cluster.py \
+          --name dask-gpu-cluster \
+          --n_workers 2 \
+          --image nvcr.io/nvidian/nemo:nightly \
+          --image_pull_secret ngc-registry \
+          --pvcs nemo-workspace:/nemo-workspace
+
+The above command uses the create_dask_cluster python code to create 2 GPU dask workers with PVCs attached to the dask-gpu-cluster.
+
+After the cluster is created, you can check if the scheduler and worker pods are running by executing:
+
+.. code-block:: bash
+
+    kubectl get pods
+
+The output will look as follows:
+
++---------------------------------------------------------+-------+---------+----------+------+
+| NAME                                                    | READY | STATUS  | RESTARTS | AGE  |
++---------------------------------------------------------+-------+---------+----------+------+
+| dask-kubernetes-operator-1720671237-6f8c579d4d-gk8pg    | 1/1   | Running | 0        | 27h  |
++---------------------------------------------------------+-------+---------+----------+------+
+| rapids-dask-default-worker-be7c9e6b19-668b8cc459-cxcwg  | 1/1   | Running | 0        | 21h  |
++---------------------------------------------------------+-------+---------+----------+------+
+| rapids-dask-default-worker-f4b5c0ff1a-66db8c4cb5-w68gd  | 1/1   | Running | 0        | 21h  |
++---------------------------------------------------------+-------+---------+----------+------+
+| rapids-dask-scheduler-5dfc446f-9tw2t                    | 1/1   | Running | 0        | 21h  |
++---------------------------------------------------------+-------+---------+----------+------+
+
+
+
+Use Kubernetes Python client library to submit NeMo-Curator jobs to the Dask scheduler:
+
+------------------------------------------------------
+
+In this method, we programmatically connect to the scheduler pod using the Kubernetes Python client library to execute the existing NeMo curator modules.
+
+This approach can be used when employing another wrapper or service to submit jobs to Dask cluster in a distributed manner.
+
+1) To execute existing NeMo curator modules in a scheduler pod from outside the EKS cluster, run the following:
```

Comment:
Can you change the numbers to use `#.` like follows
```
#. To execute existing NeMo Curator modules in a scheduler pod from outside the EKS cluster, run the following:
```
This will give nice formatting to the list.

Nit: "NeMo curator" -> "NeMo Curator"